tests/test_dataset_manager.py

```diff
@@ -406,3 +406,28 @@ def test_custom_persistent_id(simple_text_config, a_text_dataset):
             initial_tags=ALL_STANDARD_TAGS,
             dataset_split=a_text_dataset_new_col,
         )
+
+
+def test_export_proposed_actions(simple_text_config, a_text_dataset):
+    dm = DatasetSplitManager(
+        DatasetSplitName.eval,
+        simple_text_config,
+        initial_tags=ALL_STANDARD_TAGS,
+        dataset_split=a_text_dataset,
+    )
+
+    proposed_actions = {0: {"remove": True}, 2: {"relabel": True}}
+
+    dm.add_tags(proposed_actions)
+
+    path = dm.save_proposed_actions_to_csv()
+
+    name = os.path.basename(path)
+    assert name.split("_")[2:6] == ["sentiment-analysis", "eval", "proposed", "actions"]
```

Comment:
I think it wouldn't hurt to assert the whole file name, including
- the prefix `azimuth_export_`
- the date format
- the `.csv` extension

Here is a suggestion with a regular expression checking for an 8-digit date and a 6-digit time:
```suggestion
    assert re.match(
        r"azimuth_export_sentiment-analysis_eval_proposed_actions_\d{8}_\d{6}\.csv", name
    )
```

Reply:
Oh awesome! I'll change another test too!